examples/src/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { render } from 'react-dom';
-import { useModal } from '../../dist';
+import { useModal } from '../../src';
```

Comment:
Since the gh-pages build is bundled using webpack, why not import it directly from src?

Reply:
If you can deploy with this, no problem.

Reply:
We've been able to get the development server up and running, and create production builds, both with no problems!